Dockerfile

```diff
@@ -19,6 +19,6 @@ ENV RAILS_ENV=docker
 COPY . .
 
 EXPOSE 3000
-CMD ["entrypoint.sh"]
+CMD ["/rails/entrypoint.sh"]
```

Comment:
Since we set `WORKDIR /rails` previous in the Dockerfile, we don't _need_ the full path but it really doesn't matter. No need to fix

Reply:
it is necessary. or, it's has to be either in the `$PATH` or `./entrypoint.sh`